.github/actions/test/integration/build/action.yml

```
@@ -0,0 +1,126 @@
+name: build
+description: Build Bootable Virtual Machine Image
+inputs:
+  image_tag:
+    description: "Image tag to use for the build"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    # TODO https://github.com/actions/cache
+    - name: Set ENV
+      shell: bash
+      env:
+        REPO: ghcr.io/gardenlinux/gardenlinux-ccloud
+        TAG: ${{ inputs.image_tag }}
+        ESP_SIZE: 4 # GiB
+        DISK_SIZE: 8 # GiB
+      run: |
+        echo "REPO=$REPO" >> $GITHUB_ENV
+        echo "TAG=$TAG" >> $GITHUB_ENV
+        echo "ESP_SIZE=$ESP_SIZE" >> $GITHUB_ENV
+        echo "DISK_SIZE=$DISK_SIZE" >> $GITHUB_ENV
```

```suggestion
runs:
  using: "composite"
  env:
    ESP_SIZE: 4 # GiB
    DISK_SIZE: 8 # GiB
  steps:
```

Maybe let's move the env directly to `runs.env` and directly use `${{ inputs.image_tag }}` and `ghcr.io/${{ github.repository }}`